examples/android/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,7 @@
             <intent-filter>
 
                 <!-- Change this to be db- followed by your app key -->
-                <data android:scheme="db-YOUR_APP_KEY_HERE" />
+                <data android:scheme="db-yesvjbrhv075ttz" />
```

Comment:
Looks like you accidentally left your app key in. You should remove this.

Reply:
oops, obsolete key though. I've commit'd both files affected (strings.xml had it too). Is it automatically included in PR?